liip/imagine-bundle/1.8/post-install.txt

```diff
@@ -0,0 +1,17 @@
+
+  <bg=blue;fg=white>                                                            </>
+  <bg=blue;fg=white>         Getting started using </><bg=blue;fg=white;options=bold>liip/imagine-bundle</><bg=blue;fg=white>          </>
+  <bg=blue;fg=white>                                                            </>
+
+  * <fg=blue;options=bold>Configure</> <fg=blue>your transformations:</>
+    1. You <options=bold>MUST</> verify and uncomment the configuration in <comment>%ETC_DIR%/packages/imagine.yaml</>.
+    2. You <options=bold>MAY</> configure your image transformation library (<comment>gmagick</>, <comment>imagick</>, or <comment>gd</> [default]).
+    3. You <options=bold>MAY</> define custom transformation definitions under the <comment>filter_sets</> option.
+
+  * <fg=blue;options=bold>Use</> <fg=blue>your transformations:</>
+    - Using Twig: <comment>{{ imagine_filter('squared_thumbnail') }}</>
+    - Using PHP : <comment>$this->get('liip_imagine.cache.manager')->getBrowserPath('/path/to/image.jpg', 'squared_thumbnail')</>
+
+  * <fg=blue;options=bold>Read</> <fg=blue>the documentation and</> <fg=blue;options=bold>ask</> <fg=blue>for help:</>
+    - Documentation: <comment>https://symfony.com/doc/current/bundles/LiipImagineBundle</>
+    - Issue Tracker: <comment>https://github.com/liip/LiipImagineBundle/issues</>
```

Comment:
This block is way too long. It should not replace proper documentation. It should only be used to indicate the immediate next steps to make it work, if any.